css/media.css

```diff
@@ -0,0 +1,484 @@
+/*@media only screen and (min-width: 768px) {
+	.product-tile {
+		width: 50%;
+	}
+}*/
+
+/*@media only screen and (min-width: 1000px) {
+	.product-tile {
+		width: 25%;
+	}
+}*/
+
+/*@media only screen and (max-width: 1170px) {
+
+}*/
+
+@media only screen and (max-width: 1024px) {
+	.header-content ,
+	.poster-content ,
+	.product-content {
+		max-width: 1024px;
+	}
+
+	.header-container ,
+	.poster-container ,
+	.product-container ,
+	.product-items {
+		max-width: 930px;
+	}
+
+	.poster-container{
+		height: 252px;
+	}
+		
+	.product-tile {
+		max-width: 208px;
+	}
+
+	.product-tile__image {
+		max-width: 169px;
+	}
+
+	.product-tile:not(:last-child):after {
+		right: -17px;
+	}
+
+}
+
+@media only screen and (max-width: 768px) {
+	
+	html ,
+	body {
+		margin: 0;
+		padding: 0;
+	}
+
+	.header-content ,
+	.poster-content ,
+	.product-content ,
+	.header-container ,
+	.poster-container ,
+	.product-container ,
+	.product-items {
+		max-width: 768px;
+	}
+	
+	.header-container {
+		height: 111px;
+		margin: 0 44px;
+		padding: 0;
+	}
+
+	.header-container__inner {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		width: 100%;
+		height: 100%;
+		margin: 0 auto;
+	}
+
+	.header-logo__logo {
+		display: none;
+	}
+	
+	.header-logo {
+		width: 156px;
+		height: 34px;
+		margin: 0;
+	}
+
+	.header-mobile__logo {
+		display: block;
+		margin: 0;
+	}
+
+	.header-mobile__logo img {
+		width: 100%;
+	}
+
+	.header-menu__humb {
+		display: block;
+		position: relative;
+		width: 61px;
+		height: 25px;
+		cursor: pointer;
+	}
+	
+	.menu-humb_first ,
+	.menu-humb_second {
+		display: block;
+		position: absolute;
+		width: 61px;
+		height: 9px;
+		background: #000;
+		left: 0;
+	}
+	
+	.menu-humb_first {
+		top: 0;
+	}
+
+	.menu-humb_second {
+		bottom: 0;
+	}
+
+	.header-menu__list ,
+	.header-container__wrap {
+		display: none;
+	}
+
+	.poster {
+		margin-bottom: 68px;
+	}
+
+	.poster-container {
+		width: 100%;
+		height: 415px;
+	}
+
+	.poster-wrap {
+	background-position: left -127px center;
+	}
+
+	.poster-wrap__item {
+		max-width: 355px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457136)